config/config.json

```diff
@@ -46,10 +46,18 @@
         "TimeBetweenUserTypingUpdatesMilliseconds": 5000,
         "EnablePostSearch": true,
         "EnableUserTypingMessages": true,
-        "EnableChannelViewedMessages": true,
         "EnableUserStatuses": true,
+        "EnableChannelViewedMessages": true,
```

Comment:
Looks like the settings were removed from config.json. Was this intentional? 

Also, there are a few changes to config.json in your diff which might have been unintentional.